01week/rockPaperScissors.js

```diff
@@ -7,17 +7,60 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+// hand1 input rock,paper,scissors
+// check if input is valid (strings of rock, paper, scissors)
+// hand2 input rock,paper,scissors
+// check if input is valid
+// Check for a winner or tie ()
+
 
 function rockPaperScissors(hand1, hand2) {
+  // global variables and an array
+  const inputs = ['rock', 'paper', 'scissors'];
+
+  const Scrubhand1 =  hand1.toLowerCase();
+  const Scrubhand2 =  hand2.toLowerCase();
 
-  // Write code here
+  let x = false;
+  let y = false;
 
+// Check to see if user inputs are valid
+  for (let i=0; i < inputs.length; i++) {
+    if (Scrubhand1 === inputs[i]) {
+        x = true;
+      } 
+     if (Scrubhand2 === inputs[i]) {
+        y = true;
+      }
+    }
+// Outcomes of the game
+    const WinState= () => {
+      if ((Scrubhand1 == inputs[0] || Scrubhand2 == inputs[0]) && (Scrubhand1 == inputs[1] || Scrubhand2 == inputs[1])) {
```

Comment:
This wouldn't check for a win, you're checking if your lower cased  input (i.e rock) is equal to the first letter of the input.